server/utils/bankHolidays.ts

```diff
@@ -0,0 +1,27 @@
+import moment from 'moment-business-days'
+import axios from 'axios'
+import logger from '../../logger'
+
+const getBankHolidays = async () => {
+  let response = {}
+  const url = 'https://www.gov.uk/bank-holidays.json'
+  try {
+    response = await axios.get(url)
+  } catch (error) {
+    return error.response
+  }
+  return response
+}
+
+const applyBankHols = async () => {
+  return getBankHolidays().then(data => {
```

Comment:
I see the above method uses async/await and this one uses promise.then syntax. Perhaps we should all agree which one we prefer (for new projects!) and then stick with one.